### DISCUSSION TITLE
Expired time

### DISCUSSION THREAD

**Discussion by painterner** [filebrowser/filebrowser#1481] [General]
Can I set the login expired time for a long time such as one month ?

**Comment by ramiresviana** [comment]
FileBrowser does not feature such configuration.

**Reply by zilch-gg** [reply]
It closes on me when I'm uploading files. How can I keep my session up at least until all the uploads are completed?

**Reply by vendornet** [reply]
I have the same question. It seems that after some time the session expires and needs to log in again. How can I extend the session in order to keep the file uploading?

**Comment by collatdam** [comment]
It seems there was a fix as discussed here  https://github.com/filebrowser/filebrowser/discussions/3151#discussion-6550150
